Measurements/Tests.qs

```diff
@@ -349,6 +349,15 @@ namespace Quantum.Kata.Measurements {
 
         CheckSuperpositionBitstringsOneMeasurement(5, [30,14,10,6],     // [11110,01110,01010,00110]
                                                       [1,17,21,25]);    // [00001,10001,10101,11001]
+
+        CheckSuperpositionBitstringsOneMeasurement(2, [1,2],       // [01,10]
```

Comment:
This test still violates the constraint on the input for task 1.8 - bit strings 01 and 10 don't have a position in which they both are 0 or both are 1.
```suggestion
        CheckSuperpositionBitstringsOneMeasurement(2, [0,2],       // [00,10]
```